# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/ladybird/hotfix-includepath.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ladybird/Libraries/LibGfx/PaintingSurface.cpp.vanilla	2025-01-16 17:03:38.096519245 +0000
+++ ladybird/Libraries/LibGfx/PaintingSurface.cpp	2025-01-16 17:04:29.776501028 +0000
@@ -10,8 +10,8 @@
 
 #include <core/SkColorSpace.h>
 #include <core/SkSurface.h>
-#include <gpu/GrBackendSurface.h>
-#include <gpu/GrDirectContext.h>
+#include <gpu/ganesh/GrBackendSurface.h>
+#include <gpu/ganesh/GrDirectContext.h>
 #include <gpu/ganesh/SkSurfaceGanesh.h>
 
 #ifdef AK_OS_MACOS
--- ladybird/Libraries/LibGfx/SkiaBackendContext.cpp.vanilla	2025-01-16 20:00:30.002952019 +0000
+++ ladybird/Libraries/LibGfx/SkiaBackendContext.cpp	2025-01-16 20:00:41.612948672 +0000
@@ -11,7 +11,7 @@
 #include <LibGfx/SkiaBackendContext.h>
 
 #include <core/SkSurface.h>
-#include <gpu/GrDirectContext.h>
+#include <gpu/ganesh/GrDirectContext.h>
 
 #ifdef USE_VULKAN
 #    include <gpu/ganesh/vk/GrVkDirectContext.h>
